src/lib/browse/layers/areas/CensusOutputAreas.svelte

```diff
@@ -38,55 +36,10 @@
     } else {
       colorBy = "";
     }
-    if (colorBy) {
-      $map.setPaintProperty(
-        name,
-        "fill-color",
-        makeColorRamp(["get", colorBy], makeLimits(), colorScale)
-      );
-      // InteractiveLayer manages the polygon layer, but we also need to control the outline
-      $map.setLayoutProperty(outlineLayer, "visibility", "visible");
-    } else {
-      $map.setLayoutProperty(outlineLayer, "visibility", "none");
-    }
   }
 
-  overwritePmtilesSource(
-    $map,
-    name,
-    `${publicResourceBaseUrl()}/v1/${name}.pmtiles`
-  );
-
-  overwritePolygonLayer($map, {
-    id: name,
-    source: name,
-    sourceLayer: name,
-    // Initially set to a dummy value
-    color: "black",
-    opacity: hoveredToggle(0.5, 0.7),
-  });
-  overwriteLineLayer($map, {
-    id: outlineLayer,
-    source: name,
-    sourceLayer: name,
-    color: "black",
-    width: 0.5,
-  });
-
-  function tooltip(feature: MapGeoJSONFeature): string {
-    let oa = feature.properties["OA21CD"];
-    let value = feature.properties[colorBy];
-    if (colorBy == "percent_households_with_car") {
-      return `<p>${value}% of households in ${oa} have 1 or more cars</p>`;
-    } else if (colorBy == "average_cars_per_household") {
-      return `<p>Households in ${oa} have an average of ${value} cars</p>`;
-    } else {
-      return `<p>There are ${value.toLocaleString()} people per square kilometre in ${oa}</p>`;
-    }
-  }
-
-  function onClick(e: CustomEvent<MapGeoJSONFeature>) {
-    let oa = e.detail.properties["OA21CD"];
+  function onClick(e: CustomEvent<LayerClickInfo>) {
+    let oa = e.detail.features[0].properties!["OA21CD"];
```

Comment:
svelte-maplibre gives us all overlapping features. We may want to revisit the UX later and change our old approach of only ever dealing with one feature